engines/conjur_audit/contrib/gen_log.rb

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env ruby
```

Comment:
Would this file not be more appropriate in the `engines/conjur_audit/bin` directory?

Reply:
That's an excellent question. The answer is that putting it in `bin` would place the onus on us to maintain it, whereas as it is, it's just a tool script that I wrote that might be useful to somebody in the future (most probably myself...). The usual caveat of `contrib` directory that YMMV and it's not exactly maintained. 

(The name comes from the fact that often those scripts are contributed, but deemed useful and valuable enough to keep.)